### PR TITLE
Ignore errors option

### PR DESCRIPTION
**Pull request by mike1808**
I've implemented #62 to ignore errors when the key or the value cannot be evaluated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ The default message descriptors for the app's default language will be extracted
 
 - **`moduleSourceName`**: The ES6 module source name of the React Intl package. Defaults to: `"react-intl"`, but can be changed to another name/path to React Intl.
 
+- **`ignoreErros`**: Ignore errors during the messages extractions.
```
Comment (sheepsteak) [inline README.md]: Typo in \*\*\`ignoreErrors\`\*\*.

**Comment by ericf** [comment]
I'd like to understand why you need this option. If your message is defined elsewhere, then no error is thrown when the `id` is dynamic if there's no `defaultMessage`. If your message is selected dynamically, it can be defined statically via `defineMessages()`, and then use like this:

```js
const messages = defineMessages({
  foo: {
    id: 'foo',
    defaultMessage: 'Some message'
  },
  bar: {
    id: 'bar',
    defaultMessage: 'Some other message'
  }
});

export default (props) => (
  <FormattedMessage id={props.id}/>
);
```

**Comment by mike1808** [comment]
@ericf but what if I cannot define messages like that? For example, in my application I have a component that displays error message and it receives translation id and looks like this:

```jsx
export default function InputError({ id }) {
  return (
    <small className="error">
      <FormattedMessage
        id={id}
      />
    </small>
  );
}
```

And I receive the following error: 

```
SyntaxError: client/js/components/_tools/inputError/InputError.jsx: [React Intl] Messages must be statically evaluate-able for extraction.
  14 |     <small className="error">
  15 |       <FormattedMessage
> 16 |         id={id}
     |             ^
  17 |       />
  18 |     </small>
  19 |   );
```

**Comment by ericf** [comment]
If the `<FormattedMessage>` instances that it's failing on are ones **without** a `defaultMessage` prop, then I consider this a bug. I'll test this could and fix the code so it doesn't produce an error in that case. The intention is that this error only happens if a `defaultMessage` prop is defined and its value is not statically evaluatable.

**Comment by ericf** [comment]
Closing in favor of the fix in #67

**Pull request closed without merging**